### PR TITLE
Province api tests

### PR DESCRIPTION
**Pull request by acrobat**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Deprecations?   | no
| License         | MIT

### CODE REVIEW

File: src/Sylius/Bundle/CoreBundle/Tests/Controller/ProvinceApiTest.php [hunk 1]
```diff
@@ -0,0 +1,55 @@
+<?php
+
```
Comment (lchrusciel) [inline src/Sylius/Bundle/CoreBundle/Tests/Controller/ProvinceApiTest.php]: Missing license block

Comment (acrobat) [inline src/Sylius/Bundle/CoreBundle/Tests/Controller/ProvinceApiTest.php]: Fixed!

File: src/Sylius/Bundle/CoreBundle/Tests/Controller/ProvinceApiTest.php [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+{
+    /**
+     * @var array
+     */
+    private static $authorizedHeaderWithContentType = [
+        'HTTP_Authorization' => 'Bearer SampleTokenNjZkNjY2MDEwMTAzMDkxMGE0OTlhYzU3NzYyMTE0ZGQ3ODcyMDAwM2EwMDZjNDI5NDlhMDdlMQ',
+        'CONTENT_TYPE' => 'application/json',
+    ];
+
+    public function testGetProvinceAccessDeniedResponse()
```
Comment (lchrusciel) [inline src/Sylius/Bundle/CoreBundle/Tests/Controller/ProvinceApiTest.php]: In [CustomerApiTest](https://github.com/Sylius/Sylius/blob/master/src/Sylius/Bundle/CoreBundle/Tests/Controller/CustomerApiTest.php) I introduced slightly different convention of naming tests. But unfortunately we wern't consistent with this change. I'm ok with these names but we should open a RFC about this convention

Comment (acrobat) [inline src/Sylius/Bundle/CoreBundle/Tests/Controller/ProvinceApiTest.php]: Yes we weren't sure what the convention was, but as more tests had camelcase function names we assumed that camelcase would be the standard

Comment (lchrusciel) [inline src/Sylius/Bundle/CoreBundle/Tests/Controller/ProvinceApiTest.php]: Absolutely. I just wanted to highlined a problem. Anyway, good work :+1:

**Comment by pjedrzejewski** [comment]
Thanks Jeroen! :+1:

**Pull request merged**